analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/LocksReleasedAllPaths.Monitor.Conditions.vb

```diff
@@ -200,6 +200,12 @@ Namespace Monitor_Conditions
             End Set
         End Property
 
+        Public Sub Lambda()
+            Dim l = Function(value)
+                Monitor.Enter(Obj) ' Noncompliant
+                If value = 42 Then Monitor.Exit(Obj)
+                Return value
+            End Function
+        End Sub
     End Class
-
```

Comment:
_Just FYI (no change needed):_ This line should not have been removed. There should be empty line before&after `Class` and `Namespace`  lines